README.md

```diff
@@ -88,6 +88,8 @@ import url from 'file.css'
 
 ### `Useable`
 
+Means that style-loader injects the style.use() style.unuse()

```

Comment:
style-loader => \`style-loader\`

The `style-loader` injects the `styles` lazily making them useable on-demand via `style.use()/style.unuse()`

Reply:
updated in next commit